br/tests/br_parallel_restore/run.sh

```
@@ -0,0 +1,251 @@
+LOG_BACKUP_DIR="local://$TEST_DIR/log_backup"
+TABLE_COUNT=5
+CUR=$(cd `dirname $0`; pwd)
+res_file="$TEST_DIR/sql_res.$TEST_NAME.txt"
+
+. run_services
+
+# Set up trap to handle cleanup on exit
+cleanup_and_exit() {
+    # Kill all BR processes to avoid orphaned processes
+    killall -9 br.test 2>/dev/null || true
+    exit ${1:-0}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)